cmp/clipboard/ClipboardMenuItem.js

```diff
@@ -6,16 +6,27 @@
  */
 
 import {Component} from 'react';
+import {PropTypes as PT} from 'prop-types';
+import {defaultsDeep} from 'lodash';
 import {clipboardButton} from './ClipboardButton';
 import {elemFactory} from 'hoist/core';
 import {Icon} from 'hoist/icon';
 
 /**
  * Convenience wrapper for a ClipboardButton to be rendered as a Blueprint menu item.
+ * Accepts props documented below as well as any supported by Blueprint's Button or ClipboardButton cmp.
  */
 class ClipboardMenuItem extends Component {
 
-    static defaultProps = {
+    static propTypes = {
+        icon: PT.element,
+        text: PT.string,
+        cls: PT.string,
+        successMessage: PT.string,
+        style: PT.object
```

Comment:
Of these, successMessage seems to me to be the only one that's unique to this component and not something you would get on a button. I can also see including icon and text to help make it clear what the two primary props are you will almost certainly be specifying, but cls and style are generally on everything.  Does that make sense / seem consistent enough?

Reply:
agree we need to leave style and cls off